raiden/blockchain/events.py

```diff
@@ -396,6 +399,13 @@ def fetch_logs_in_batch(self, target_block_number: BlockNumber) -> PollResult:
         Ethereum node to process the request, which will result in read
         timeouts (issue #3558).
 
+        The block batch size is adjusted dynamically based on the request
+        processing duration (see ``_query_and_track()``, issue #5538).
```

Comment:
We usually use single backticks in comments, don't we?

Reply:
Yeah we're not consistent. But since this is a docstring and they are supposed to be RST two backticks is the correct syntax.